### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by 733737**
Fixed typos but did not change the meaning of the text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
-**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), to decouple its reliance on the copyrighted material from the original game and allow anyone to play the game for free.
+**OpenGraphics** is a project established to replace the base graphics of Rollercoaster Tycoon 2. The aim is to create a set of freely-useable graphics that can be used with [OpenRCT2](https://github.com/OpenRCT2/OpenRCT2), in order to decouple its reliance on the copyrighted material from the original game, and to allow anyone to play the game for free.
 
 ## Getting Started
 
-We're in the process of setting up pipelines and deciding on the direction of OpenGraphics. If you have experience with art or tools development than consider checking out our discussions about establishing the rendering process:
+We're in the process of setting up pipelines and deciding on the direction of OpenGraphics... If you have experience with art or tool development then consider checking out our discussions about establishing the rendering process:
```
Comment (Broxzier) [inline README.md:7]: The elipsis shouldn't be used in descriptions.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 https://github.com/OpenRCT2/OpenGraphics/issues/24
 
-You can already start helping out by creating 3D models for scenery objects. If possible, try to use Blender 2.7. Check out [the spreadsheet](https://drive.google.com/open?id=1ljVeYxp8ijj5z4VuSa6Xo1-_p3wACdQCRudkvkLGApc) to see what has already been made. Just mark an object as WIP, and add your name to the author field.
+You can already start helping out by creating 3D models for scenery objects. If possible, you should use Blender 2.7. Check out [the spreadsheet](https://drive.google.com/open?id=1ljVeYxp8ijj5z4VuSa6Xo1-_p3wACdQCRudkvkLGApc) to see what has already been made.
+Mark an object as WIP and add your name to the author field to contribute.
```
Comment (Broxzier) [inline README.md:11]: Why did you choose to remove the oxford comma here, while you added one to the first paragraph?

**Review by Broxzier — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (pizza2004) [reply README.md:11]: That's not an Oxford comma it's just grammatically incorrect.  An Oxford comma is just a comma used after the penultimate item in a list of 3 or more items.  Even still I think moving "to contribute" to be beginning of the sentence would make it sound less stilted.

File: README.md (hunk 1 above)
Comment (733737) [reply README.md]: I just changed the issues.

**Review by oli414 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
+You can already start helping out by creating 3D models for scenery objects. If possible, you should use Blender 2.7. Check out [the spreadsheet](https://drive.google.com/open?id=1ljVeYxp8ijj5z4VuSa6Xo1-_p3wACdQCRudkvkLGApc), to see what has already been made.
+To contribute, simply mark an object as WIP and add your name to the author field.
 
 ## Chat
 
 You can get in touch with us via the OpenRCT2 Discord server. There is a dedicated OpenGraphics channel that can be used to discuss this sub-project.
 
 - Invitation link: https://discord.gg/uNzSmAj
-- [![](https://img.shields.io/discord/264137540670324737?label=OpenRCT2%2Fgraphics)](https://discordapp.com/channels/264137540670324737/691752238057783356)
+- [![](https://img.shields.io/discord/264137540670324737?label=OpenRCT2%2Fgraphics)](https://discordapp.com/channels/264137540670324737/691752238057783356),
```
Comment (Gymnasiast) [inline README.md:18]: Why this trailing comma?

**Pull request merged**